login/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import authenticate, login
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import logout
+
+def home_view(request):
```

Comment:
can we also make the views class based instead of method based (like in the given apps from labs)